### ISSUE TITLE
Guided Learning Paths

### ISSUE THREAD

**Issue by srbhr** [Git-Commit-Show/100-Days-of-Code#233]
@adityajoshi-08 As discussed, we need to create guided learning paths for "Software Development" so that people can get something of this 100 Days Free Coding Bootcamp on their own.

- Software Design (Design Patterns)
- Data Structures
- Algorithms
- System Design
- Databases
Anything else that you can think of?

**Comment by adityajoshi-08** [comment]
Should we add resources in this repo as separate folder?

**Comment by srbhr** [comment]
Yes @adityajoshi-08 they're mostly markdowns but each one having what to do, and from where. It'll take time, so let it be a WIP throughout!

**Comment by adityajoshi-08** [comment]
Yes, but we shall start adding resources. Let us take this way.
We should focus that the resources are complete on the first place and are then merged into the repo.

**Comment by adityajoshi-08** [comment]
I shall start making a resource markdown for web development as I already have a list of some good ones.

**Comment by srbhr** [comment]
@adityajoshi-08 Still waiting for you PR

**Comment by kibetamos** [comment]
[@saurabhraidev] did you get ?

**Comment by saurabhraidev** [comment]
@kibetamos no not yet
